test/graphql/Applicant/mutations/updateCurrentApplicant.test.ts

```diff
@@ -139,6 +139,22 @@ describe("updateCurrentApplicant", () => {
   });
 
   describe("Errors", () => {
+    it("returns an error if approvedYearCount is a number for a graduated", async () => {
```

Comment:
returns an error if isGraduate and approvedYearCount are defined at the same time